src/components/ui/TextArea/TextArea.tsx

```diff
@@ -8,11 +8,16 @@ import InputDefaultClass from 'antd/lib/input/TextArea'
 import styles from './TextArea.less'
 import ReadOnlyField from '../ReadOnlyField/ReadOnlyField'
 import {TextAreaProps as AntdTextAreaProps} from 'antd/lib/input/TextArea'
+import {WidgetField} from 'interfaces/widget'
 
 type AdditionalAntdTextAreaProps = Partial<Omit<AntdTextAreaProps, 'onChange'>>
 export interface TextAreaProps extends AdditionalAntdTextAreaProps {
+    cursor?:string,
+    widgetName?: string,
+    metaError?: string,
+    meta?: WidgetField, // TODO 2.0.0 must be required?
```

Comment:
It unnecessary binds a regular UI component to a widget context and narrows its usage, i.e. you'll no longer be able to use this component in other parts of the application.  
UI component shouldn't care about widgets, cursors and meta.